exporter/sentryexporter/ratelimit.go

```diff
@@ -0,0 +1,113 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sentryexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sentryexporter"
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	internalrl "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sentryexporter/internal/ratelimit"
+)
+
+var errRateLimited = errors.New("error sending data, rate-limited")
+
+const defaultRetryAfter = 60 * time.Second
```

Comment:
l: This is duplicate with the value in `exporter/sentryexporter/internal/ratelimit/retry_after.go`. Should we use one constant?